Source/WebCore/page/InteractionRegion.cpp

```diff
@@ -139,11 +136,25 @@ std::optional<InteractionRegion> interactionRegionForRenderedRegion(RenderObject
         return std::nullopt;
 
     // FIXME: Consider also allowing elements that only receive touch events.
-    if (!renderer.style().eventListenerRegionTypes().contains(EventListenerRegionType::MouseClick))
+    auto cursor = cursorTypeForElement(*element);
+    if (!renderer.style().eventListenerRegionTypes().contains(EventListenerRegionType::MouseClick)
```

Comment:
Maybe give this set of conditions a local with a meaningful name so it's easier to follow that this is the "non-interactive" path?